### ISSUE TITLE
FS1 Support

### ISSUE THREAD

**Issue by guytz72** [christianrehn/CombineTest#6]
Is there any way to add support for FS1 software?  I have a GC2 with FS1 and have had a look around in the program directories but can't see and equivalent of the LastShot.csv.  What there is however is the ability to export fitting data to csv.  An example of this is attached, you have to name the file yourself so it will change, as will the name of the clubs in the "Club" column. 

Something I could see working is this (assuming yards for this):

In FS1:
1. Create clubs named 60,70,80,90,100,120,140,160,180,DRIVE to correspond to the combine distances.  You only have to do this once and they will be saved.
2. Start a club fitting, quick start
3. Hit 6 shots to each of the distances, selecting the "club" from 1 above  that relates to the target distance

4. Select save csv and give and appropriate name

In the CombineTest App
1.  Click a new option, "Import from FS1"
2. Browse to the file created in no 4 above

Ideally the app would cater for hitting the shots in any order but this is a nice to have (so you can hit 3 shots to each distance then repeat as you do on TM Combine).  So some sort of lookup based on target distance and the club in the csv.

[test.CSV](https://github.com/christianrehn/CombineTest/files/12805417/test.CSV)

**Comment by christianrehn** [comment]
I have an idea that might be simple to implement:
CombineTest saves all data of a session in json format in order to show the results later. 
Maybe I could write some code to import your CSV file like a session (which I think is the correct interpretation of the data because your CSV data is the result of a training session). All values except target distance are already in the csv file. If you name your clubs as suggested above I could interpret the club names as the target distances.
During the CSV import you only have to select a drillConfiguration interactively and CombineTest should be able to calculate all the values for the shots like distance to hole, combine test points, shots gained, ...

If I find some time within the next days/weeks I will try to implement the CSV import.

**Comment by guytz72** [comment]
Yes, that sounds like a great solution!  I think, but I'd have to check that this would also work from the Performance Testing iOS app as you can email that fitting data.  That app is very buggy and I keep having to delete and reinstall it but I'll try and get it working again and check the output of that file.

Thanks for your time with this!
